src/macad_gym/core/maps/nav_utils.py

```diff
@@ -384,6 +382,12 @@ def draw(self):
                     color=carla.Color(0, 255, 0),
                     thickness=0.5)
 
+    def plot(self):
+        import matplotlib.pyplot as plt
+        plt.scatter([i[0].transform.location.x for i in self.path], [i[0].transform.location.y for i in self.path])
+        plt.gca().invert_yaxis()
+        plt.show()
+
```

Comment:
Where is this `plot()` method used or is useful?

Reply:
gives a plot of trajectories instead of using the drawaing method in the carla environment. Is useful for visualizing the path towards the target. Anyway it can be removed